lib/_http_outgoing.js

```diff
@@ -66,7 +66,8 @@ const {
     ERR_STREAM_ALREADY_FINISHED,
     ERR_STREAM_WRITE_AFTER_END,
     ERR_STREAM_NULL_VALUES,
-    ERR_STREAM_DESTROYED
+    ERR_STREAM_DESTROYED,
+    ERR_CONTENT_LENGTH_HEADER
```

Comment:
```suggestion
    ERR_CONTENT_LENGTH_MISMATCH
```

Reply:
nit: Could this list also be sorted in alphabetical order?

Reply:
Ok, thanks for the feedback, would make the changes